src/makeRobotAccountant.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return (num1) => {
```

Comment:
```suggestion
  return (number1) => {
```
in this task not critical, but in general - avoid cutting of names + use descriptive ones